### PR TITLE
Br1 first feature for this project 

### PR DESCRIPTION
**Pull request by EllenZhangQ**
Add new feature
1 new file add
2, modified file

### CODE REVIEW

**Comment by EllenZhangQ** [comment]
good job

File: newFeature.txt
Comment (EllenQingZhang) [inline newFeature.txt:1]: good

**Review by EllenQingZhang — APPROVED** [review]
go ahead!

**Review by EllenQingZhang — APPROVED** [review]
Well done!

**Review by EllenQingZhang — APPROVED** [review]

**Review by EllenQingZhang — APPROVED** [review]
Approve

**Pull request merged**